### ISSUE TITLE
FEATURE: Allow input of date with keyboard in DateTime Editor

### ISSUE THREAD

**Issue by crydotsnake** [neos/neos-ui#3394]
### Description

When working with the `DateTime` Editor, it can sometimes be annoying to set the date using only the arrows. It would be a significant improvement if I had the possibility to set the date also via the keyboard input.

![SCR-20230223-flm](https://user-images.githubusercontent.com/39345336/220981777-ab252579-efa1-4e35-834b-1310d9089e3b.png)

**Comment by mhsdesign** [comment]
Yes.

Infact this is only the least of our problems as the UI is not optimized at all for pro/power users (maybe because they dont exist?) but there are a lot of keyboard related issue (thats why i introduced the label and the umbrella issue:  https://github.com/neos/neos-ui/issues/3100)

**Comment by JamesAlias** [comment]
@mhsdesign I agree. The accessibility of Neos is not in a great spot right now 🙈

**Comment by mficzel** [comment]
Added https://github.com/neos/neos-ui/issues/3421 (remove separate "apply" button from DatePicker) which is kindoff related and could be attacked together
